test/library/standard/FileSystem/chmod/chmod_noprivilege.execopts

```diff
@@ -0,0 +1 @@
+--original=0o755
```

Comment:
(ignore the old comment, I misread things)

It looks like this file is generated.  I think you both:
A) don't want to commit it at all, and
B) want to add it to the .gitignore file